### DISCUSSION TITLE
pass router params to not-found.js

### DISCUSSION THREAD

**Discussion by mildfuzz** [vercel/next.js#48395] [Ideas]
### Goals

1. Give access to router params as props to not-found.js default export 
2. Enable server render that incorporates information from route 

### Background

At the moment, default export in 'page.js' has access to router params as props within a server render. 

### Proposal

It would be useful to have this available to the 'not-found.js'

**Comment by steveluscher** [comment]
Duplicate of https://github.com/vercel/next.js/discussions/43179.
